HfsChargesContainer/Helpers/RuntimeEnvVarsHandler.cs

```diff
@@ -0,0 +1,68 @@
+using Amazon;
+using Amazon.SimpleSystemsManagement;
+using Amazon.SimpleSystemsManagement.Model;
+using HfsChargesContainer.Helpers.Interfaces;
+using HfsChargesContainer.Options;
+
+namespace HfsChargesContainer.Helpers
+{
+    public class RuntimeEnvVarsHandler : IRuntimeEnvVarsHandler
+    {
+        private readonly IAmazonSimpleSystemsManagement _ssmClient;
+        private readonly Dictionary<string, string> _ssmToAppNameLookup;
+
+        public RuntimeEnvVarsHandler(
+            string environment,
+            IAmazonSimpleSystemsManagement? ssmClient = null,
```

Comment:
why are these 2 parameters defaulted to null in the constructor?
The Dependency Injection container should create instances of these.

Reply:
It's because I made it so RuntimeEnvVarsHandler news up the ssm client itself rather than depends on startup to provide it. This way the constructor doesn't need the arguments at all... except when testing... when testing we need to provide a mock implementation of the client, so a parameter slot is needed, which has to be optional... hence it defaults to null.

Reply:
I don't mind making the Startup provide the client though - don't have a preference. So I can change it if you want.

Reply:
Update: We initially agreed that it would be good to changes this, however it turned out that changing this complicates the testing of startup behaviour. So we decided to leave this as is.